### PR TITLE
cli: basic interface

### PR DESCRIPTION
**Pull request by omelkonian**
* Adds basic CLI, including creating new sequences and
   asking for next counters on existing sequences.

 * Adds tests for coverage.

Signed-off-by: Orestis Melkonian <melkon.or@gmail.com>

### CODE REVIEW

File: tests/test_cli.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    assert_success(author_1(), 'John.Tester.1')
+    assert_success(author_1(), 'John.Tester.3')
+    assert_success(author_1(), 'John.Tester.5')
+
+    author_2 = partial(runner.invoke, sequences,
+                       ['next', 'AUTH', 'author=Tom.Phake'])
+    assert_success(author_2(), 'Tom.Phake.1')
+    assert_success(author_2(), 'Tom.Phake.3')
+    assert_success(author_2(), 'Tom.Phake.5')
+
```
Comment (egabancho) [inline tests/test_cli.py]: I guess we don't need all this blanks 😉

**Review by egabancho — CHANGES_REQUESTED** [review]
@omelkonian What do you think about updating the docs with a CLI usage section?

File: invenio_sequencegenerator/ext.py [hunk 2]
```diff
@@ -40,3 +42,6 @@ def init_app(self, app):
         app.register_blueprint(Blueprint(
             'invenio_sequencegenerator', __name__, template_folder='templates')
         )
+
+        # Add CLI
+        app.cli.add_command(sequences)
```
Comment (jirikuncar) [inline invenio_sequencegenerator/ext.py]: You can use `flask.commands` entry point (example https://github.com/inveniosoftware/invenio-access/blob/master/setup.py#L107-L109) and then you can revert all changes in this file.

**Review by egabancho — APPROVED** [review]

File: tests/test_cli.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    """Test CLI for creating template definitions and asking for counters."""
+    runner = CliRunner()
+    run = partial(runner.invoke, sequences, obj=script_info)
+
+    def assert_success(res, expected):
+        assert res.exit_code == 0
+        assert res.exception is None
+        assert expected in res.output
+
+    #
```
Comment (jirikuncar) [inline tests/test_cli.py]: Playlists and authors should be separate test cases.

**Pull request merged**